tests/sample/assemblies/test_repeating_multilayer.py

```diff
@@ -37,8 +37,8 @@ def test_from_pars(self):
         k = Material.from_pars(0.487, 0.000, 'Potassium')
         p = Layer.from_pars(m, 5.0, 2.0, 'thinBoron')
         q = Layer.from_pars(k, 50.0, 1.0, 'thickPotassium')
-        l = LayerCollection.from_pars(p, q, name='twoLayer')
-        o = RepeatingMultilayer.from_pars(l, 2.0, 'twoLayerItem')
+        layers = LayerCollection.from_pars(p, q, name='twoLayer')
```

Comment:
considering all other defaults are single letters, maybe `layers` should be `l` after all?

Reply:
It was changed because ruff complained that the `l` was an `Ambiguous variable name`